store/src/main/java/org/apache/rocketmq/store/DefaultMessageStore.java

```diff
@@ -849,7 +849,11 @@ public GetMessageResult getMessage(final String group, final String topic, final
                                 continue;
                             }
 
-                            this.storeStatsService.getGetMessageTransferredMsgCount().add(cqUnit.getBatchNum());
+                            if (topicConfig.isPresent()) {
+                                if (!topicConfig.get().getTopicName().contains(TopicValidator.RMQ_SYS_REVIVE_LOG)) {
```

Comment:
Store metrics should not exclude specified topics. It's better to add a new counter in BrokerStatsManager to calculate the message transfer count excluding all system topics.

Reply:
IMO, It will broke the current structure of StoreStatsService if moving the getMessageTranferredMsgCount to BrokerStatsManager,

because not only is getMessageTransferredMsgCount in there but also other metrics is there,

whether moving the other metrics if moving getMessageTransferredMsgCount to BrokerStatsManager?

Reply:
I mean add a new counter in BrokerStatsManager and do not remove the current counter. The new counter only records the throughput of the user's topic. This is more general and semantically clear.

Reply:
Get it, I will polish it